config/routes.rb

```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
-
   devise_for :users
 
   resources :features
   resources :plans
+  resources :transactions
+
+  resources :subscriptions do
+    post :subscribe
+    delete :unsubscribe
```

Comment:
why can't we use subscriptions delete resource for this purpose?

Reply:
Shifted to plans controller